sdk/energyservices/Azure.ResourceManager.EnergyServices/src/autorest.md

```diff
@@ -0,0 +1,48 @@
+# Generated code configuration
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: EnergyServices
+title: EnergyServices
+namespace: Azure.ResourceManager.EnergyServices
```

Comment:
Is this namespace reviewed by archboard?

Reply:
not yet. is that needed?

Reply:
not required for beta but mandatory for GA

Reply:
thanks for confirming.